tools/installer/full-node.sh

```diff
@@ -0,0 +1,239 @@
+#!/bin/bash
```

Comment:
I'm not going to review this bash code, but I have a question and a request:

1. Request - Can you add a make target to trigger this?
2. Question - Should this work on my local machine if I just run it as is?

Reply:
This script is intended to be downloaded and ran on the machine (something like https://get.docker.com/). As a result:
1. When would we need a make target for it if we should only run the script on something like VPS/Dedicated Server?
2. Negative, the script is not meant to run in a non-linux environment, and I have not tested WSL2.